src/split-props.js

```diff
@@ -13,6 +13,10 @@ export default function splitProps(
   },
   {propsAreCssOverrides, rootEl, filterProps, forwardProps},
 ) {
+  // forward innerRef if user wishes to do so
+  if (innerRef !== undefined && forwardProps.indexOf('innerRef') !== -1) {
```

Comment:
what's the point of checking `innerRef`'s value here?

Reply:
Performance. There's no need to loop through the array if `innerRef` is empty anyway.
It's in glamorous' "hot path", but I'm not sure it's really that critical. Happy to leave it out.

Reply:
I conveniently leave decisions concerning optimizations vs longer code to @kentcdodds :)

Reply:
I'm fine with it as-is :+1:

Thanks!